packages/node/src/services.ts

```diff
@@ -202,3 +204,99 @@ export function confirmLocalFirebaseConfigurationEnvVars() {
     );
   }
 }
+
+export interface PostgresConfiguration {
+  user: string;
+  host: string;
+  database: string;
+  password: string;
+  port: number;
+}
+
+export const POSTGRES_CONFIGURATION_ENV_KEYS = {
+  user: "POSTGRES_USER",
+  host: "POSTGRES_HOST",
+  database: "POSTGRES_DATABASE",
+  password: "POSTGRES_PASSWORD",
+  port: "POSTGRES_PORT"
+};
+
+export const EMPTY_POSTGRES_CONFIG: PostgresConfiguration = {
+  user: "",
+  host: "",
+  database: "",
+  password: "",
+  port: 0
+};
+
+export class PostgresServiceFactory {
+  private client: Client;
+
+  constructor(configuration: PostgresConfiguration) {
+    this.client = new Client(configuration);
+  }
+
+  static async connect(host: string, port: number) {
+    new PostgresServiceFactory({
+      ...EMPTY_POSTGRES_CONFIG,
+      host,
+      port
+    });
+  }
+
+  async createStoreService(storeServiceKey: string): Promise<IStoreService> {
+    await this.client.connect();
+    console.log("Connected to Postgres");
+    return new PostgresStoreService(this.client, storeServiceKey);
+  }
+}
+
+class PostgresStoreService implements IStoreService {
+  constructor(
+    private readonly pgClient: Client,
+    private readonly storeServiceKey: string
+  ) {}
+
+  async get(key: string): Promise<any> {
```

Comment:
@rhlsthrm I think we can type more accurately here by returning `Promise<StringKeyValue>`:

```ts
type StringKeyValue = { [key: string]: StringKeyValue };

async get(key: string): Promise<StringKeyValue>
```

We're trying to stear clear of `any` as much as possible.